docs/BuildBrainFlow.rst

```diff
@@ -101,10 +101,12 @@ BrainFlow is a registered package in the Julia general registry, so it can be in
     Example: ::
 
         import Pkg
-		Pkg.add("BrainFlow")
+        Pkg.add("BrainFlow")
         
 When using BrainFlow for the first time in Julia, the BrainFlow artifact containing the compiled BrainFlow libraries will be downloaded.
 
+BrainFlow developers or expert users may also place compiled libraries into /lib inside the julia package root folder. These will take precedence over the artifact.
```

Comment:
lets change this sentence.

When you compile brainflow from the source cmake automatically places all libs there you dont need to copypaste libs

Reply:
Sure it's in the CI, but I am also using this for local development. And you could switch manually to the OpenMP variant right?

Reply:
if compiled locally with openmp enabled,  will be improved in next version